.travis.yml

```diff
@@ -23,6 +23,7 @@ env:
     - SFDX_AUTOUPDATE_DISABLE=true
     - SFDX_USE_GENERIC_UNIX_KEYCHAIN=true
     - SFDX_DOMAIN_RETRY=300
+    - CODE_TESTS_WORKSPACE=$TRAVIS_BUILD_DIR/packages/system-tests/sfdx-simple
```

Comment:
And if you need this, then it must be because of your new tests. And, if that is so, then your tests are structured wrongly if you need to set a global variable.

What happens if you there are other tests that need different workspace?